### ISSUE TITLE
Non-Trans job and other Job exp gain not obtaining too much

### ISSUE THREAD

**Issue by AsurielRO** [rathena/rathena#8902]
### rAthena Hash

 84b4832

### Client Date

20220406

### Server Mode

Renewal

### Result

For the test use Ninja Job and try to reach at least lvl 50.
1. You will notice that job exp gain even using a job exp book, does not give too much.
2. You can levelup your main level more than job exp.

In my test I set the exp rates to 30x30x3

### Expected Result

Must gain as normal that follows the server rates like for example player at level 2 with level 2 job level as well or maybe more based on the setup.

### How to Reproduce

--

### Official Information

--

**Comment by Atemo** [comment]
What is your "Expected Result" based on?

**Comment by AsurielRO** [comment]
> What is your "Expected Result" based on?

The expected result is that the job experience gained would be normal like for the first job and other jobs.

**Comment by Lemongrass3110** [comment]
Maybe use `@showexp` and give some examples with concrete numbers.
Right now (for me at least) it is hard to follow what your problem is or what you want to have differently.

**Comment by Playtester** [comment]
Also notably Ninja / Gunslinger classes have their own Exp table, so already the expected result is incorrect. For example Ninja / Gunslinger need 200 Exp from Job 1 to Job 2 but 1st jobs need 60 Exp from Job 1 to Job 2.

**Comment by Lemongrass3110** [comment]
Unable to reproduce and user did not deliver any samples for a month.
